### ISSUE TITLE
Mod wont load

### ISSUE THREAD

**Issue by ebkboogie** [loki79uk/FS25_LumberJack#15]
The mod only loads successfully about 1 in 10 tries. I can edit settings and key binds, but the key bind to toggle super strength wont appear in the info menu as well as all mod's functions won't work such as super strength and stump removal. I've tested all versions, checked logs for issues, and even tried loading a map with just Lumberjack enabled, but it still doesn't work. Any fixes? Thanks in advance.

**Comment by loki79uk** [comment]
Could you have corrupted the settings XML file? Try deleting it completely and launch the game.  If that doesn't work, can you attach a log file from when it doesn't load?

**Comment by loki79uk** [comment]
Sorry for delay, I've been busy with UAL.  I can only guess this muse be a key binding issue?  Do other features still work like bush removal?  If you use the lastest version then you can change lumberjack settings in the menu.  Turn on debugging there, restart and there will be some extra stuff in the log.

**Comment by ebkboogie** [comment]
[log.txt](https://github.com/user-attachments/files/17956047/log.txt)
none of the functions work however the log now shows "2024-11-29 18:37 [LumberJack] LUMBERJACK: ADDING ACTION EVENT
2024-11-29 18:37 [LumberJack]  success = false"
not sure why tho

**Comment by loki79uk** [comment]
I need something more to go on that just "it doesn't work".   Are you using the latest version?  is the menu showing?

**Comment by ebkboogie** [comment]
I can see the menu in settings, all features off the mod doesn't work, cant see super strength key bind but can still set it in the controls menu. Using version of 1.0.0.3.

**Comment by ebkboogie** [comment]
Yep cutting trees on unowned land as well as deleting stumps works. But I still cant de limb or enable super strength.

**Comment by ebkboogie** [comment]
This issue is actually so confusing bc i used power tools super strength and that works fine, and it also allows for me to completely delete trees with it.

**Comment by loki79uk** [comment]
> This issue is actually so confusing bc i used power tools super strength and that works fine, and it also allows for me to completely delete trees with it.

That actually does make more sense now.  It must be a conflicting key binding.  Try resetting your key bindings. (Back up the file first)

Lumberjack detects the use of the console command or any other mod using the console command to toggle super strength, so you don't have to use the bound key.
